eras/babbage/impl/src/Cardano/Ledger/Babbage/Rules/Utxo.hs

```diff
@@ -471,7 +465,6 @@ instance
       work (FromAlonzoUtxoFail x) = Sum FromAlonzoUtxoFail 1 !> To x
       work (FromAlonzoUtxowFail x) = Sum FromAlonzoUtxowFail 2 !> To x
       work (UnequalCollateralReturn c1 c2) = Sum UnequalCollateralReturn 3 !> To c1 !> To c2
-      work (DanglingWitnessDataHash x) = Sum DanglingWitnessDataHash 4 !> To x
```

Comment:
:thinking: I'm wondering if it makes sense to leave a comment explaining why we've mysteriously skipped `4`...

Reply:
Yes absolutely

Reply:
thank you @lehins for pointing out to me that this is Babbage code, so it's totally fine to change the serialization (ie not skip 4)! sorry for the confusion